plugins/pytorch/CMakeLists.txt

```diff
@@ -70,3 +69,29 @@ if( VIAME_ENABLE_PYTORCH-MDNET )
     mdnet_tracker_process )
 
 endif()
+
+if( VIAME_ENABLE_PYTORCH-CUTLER )
```

Comment:
These mmdet* processes are not meant for just running CUTLER and should not be quarantined into requiring it, I haven't looked at the implementation changes of these files yet, but my guess is you're breaking existing functionality in there (the ability to run mmdet models without CUTLER). If the CUTLER option requires a lot of extra dependencies, you can either add a flag in the file itself to enable it's use, or you should make new .py files alongside these specifically for running it and quarantine them off via the ENABLE-CUTLER flag